tests/test_dask_image/test_ndfilters/test__conv.py

```diff
@@ -154,3 +154,35 @@ def test_convolutions_compare(sp_func,
             d, weights, origin=origin
         )
     )
+@pytest.mark.parametrize(
```

Comment:
Love how thorough this test is using pytest parametrize!

(For a brief moment I thought this test might need a 1D instead of 2D example, but on reflection I don't think that matters)

Reply:
I saw that you previously said that the test works even without my changes. I'm not sure that I understand how it is possible. I tested myself on the exact same values than in the test and `mode = "wrap"` shows a discrepancy between `scipy.ndimage` and `dask_image` that dissappears after adding the changes I made.

Reply:
I was trying to do two things at once and made a silly mistake earlier. It's ok!